mpa/modules/hooks/__init__.py

```diff
@@ -18,3 +18,4 @@
 from . import early_stopping_hook
 from . import progress_update_hook
 from . import logger_replace_hook
+from . import save_initial_weight_hook
```

Comment:
There seems to be a flake8 issue.

Reply:
You mean f-srting issue in save_initial_weight_hook, right?